puppet/modules/web/manifests/vhost/yum.pp

```diff
@@ -27,11 +29,22 @@
     },
   ]
 
+  if $setup_receiver {
+    secure_ssh::receiver_setup { $user:
+      user           => $user,
+      foreman_search => 'host ~ node*.jenkins.osuosl.theforeman.org and (name = external_ip4 or name = external_ip6)',
+      allowed_ips    => ['192.168.122.1'],
```

Comment:
probably not ;)

Reply:
Technically this will be unused in production when Foreman API credentials are present, but you're right; that was my test code :)